src/test/java/io/cdap/plugin/sfmc/sink/DataExtensionInfoTest.java

```diff
@@ -0,0 +1,119 @@
+    column.setId("121");
+    column.setDescription("newCustomer");
+    column.setLength(22);
+    Date d = new Date(2022, 02, 9);
+    column.setCreatedDate(d);
+    column.setDefaultValue("default");
+    column.setIsPrimaryKey(true);
+    column.setIsRequired(true);
+    column.setKey("key");
+    column.setModifiedDate(d);
+    column.setName("name");
+    column.setModified("date", false);
+    column.setPrecision(0);
+    column.setScale(100);
+    columnList.add(column);
+    Map<String, ETDataExtensionColumn> columns = new HashMap<>();
+    columns.put("tableName", column);
+    DataExtensionInfo info =
+      new DataExtensionInfoBuilder().setColumnList(columnList).setColumns(columns).setExternalKey(TEST_EXTERNALKEY).
+        build();
+    Assert.assertEquals(columnList.size(), 1);
+    Assert.assertFalse(info.getColumnList().isEmpty());
+  }
+
+  @Test
+  public void testHasColumns() {
+    List<ETDataExtensionColumn> columnList = new ArrayList<>();
+    ETDataExtensionColumn column = new ETDataExtensionColumn();
+    column.setId("121");
+    column.setDescription("newCustomer");
+    column.setLength(22);
+    Date d = new Date(2022, 02, 9);
+    column.setCreatedDate(d);
+    column.setDefaultValue("default");
+    column.setIsPrimaryKey(true);
+    column.setIsRequired(true);
+    column.setKey("key");
+    column.setModifiedDate(d);
+    column.setName("name");
+    column.setModified("date", false);
+    column.setPrecision(0);
+    column.setScale(100);
+    columnList.add(column);
+    Map<String, ETDataExtensionColumn> columns = new HashMap<>();
+    columns.put("tableName", column);
+    DataExtensionInfo info =
+      new DataExtensionInfoBuilder().setColumnList(columnList).setColumns(columns).setExternalKey(TEST_EXTERNALKEY)
+        .build();
+    Assert.assertEquals(columns.size(), 1);
+    Assert.assertFalse(info.hasColumn("tableName"));
+  }
+
+  @Test
+  public void testGetColumn() {
+    List<ETDataExtensionColumn> columnList = new ArrayList<>();
+    ETDataExtensionColumn column = new ETDataExtensionColumn();
+    column.setId("121");
+    column.setDescription("newCustomer");
+    column.setLength(22);
+    Date d = new Date(2022, 02, 9);
+    column.setCreatedDate(d);
+    column.setDefaultValue("default");
+    column.setIsPrimaryKey(true);
+    column.setIsRequired(true);
+    column.setKey("key");
+    column.setModifiedDate(d);
+    column.setName("name");
+    column.setModified("date", false);
+    column.setPrecision(0);
+    column.setScale(100);
+    columnList.add(column);
+    Map<String, ETDataExtensionColumn> columns = new HashMap<>();
+    columns.put("tableName", column);
+    DataExtensionInfo info =
+      new DataExtensionInfoBuilder().setColumnList(columnList).setColumns(columns).setExternalKey(TEST_EXTERNALKEY)
+        .build();
+    Assert.assertEquals("name", column.getName());
+    Assert.assertFalse(info.getColumn("name").getKey().isEmpty());
```

Comment:
assert the actual value of the key, not just check for non-empty

Reply:
Done

Reply:
remove above Asser.assertFalse() as it's redundant (you're asserting on value of the key in the next line)

Reply:
Removed